internal/cmd/schema-registry/command_cluster_delete.go

```diff
@@ -41,24 +40,26 @@ func (c *command) newClusterDeleteCommand() *cobra.Command {
 }
 
 func (c *command) clusterDelete(cmd *cobra.Command, _ []string) error {
-	ctx := context.Background()
-
 	environmentId, err := c.EnvironmentId()
 	if err != nil {
 		return err
 	}
 
-	cluster, err := c.Context.FetchSchemaRegistryByEnvironmentId(ctx, environmentId)
+	clusterList, _, err := c.V2Client.GetSchemaRegistryClusterByEnvironment(environmentId)
```

Comment:
The second argument can contain useful error details. Take a look at some other SDKs for how they do this.

Reply:
https://github.com/confluentinc/cli/blob/main/internal/pkg/ccloudv2/cdx.go is a good example.

Reply:
Based on the function name, it looks like this is returning one cluster, yes?
```suggestion
	cluster, _, err := c.V2Client.GetSchemaRegistryClusterByEnvironment(environmentId)
```